compiler/rustc_resolve/src/lib.rs

```diff
@@ -456,28 +456,36 @@ struct BindingKey {
 type Resolutions<'a> = RefCell<FxIndexMap<BindingKey, &'a RefCell<NameResolution<'a>>>>;
 
 /// One node in the tree of modules.
+///
+/// Note that "module" is a loose term here; it does not necessarily mean
```

Comment:
It's a precise term, it just means a different thing.
Module in this context is a container for named items processed early (and thus excluding items that are resolved in type-relative way in type checker).
So it refers either to `mod`, or crate root, or `enum`, or `trait`, or curly-braced block with statements.

Reply:
Updated.